From 950e94b0ad06eaca2227956559334c511e3064f2 Mon Sep 17 00:00:00 2001
From: Kristian Fiskerstrand <kristian.fiskerstrand@sumptuouscapital.com>
Date: Wed, 24 Feb 2016 10:15:47 +0100
Subject: [PATCH] templates/lxc-gentoo.in: Fix download issue due to updated
 pointer

templates/lxc-gentoo.in downloads a pointer to the latest
stage3 release, e.g
http://distfiles.gentoo.org/releases/amd64/autobuilds/latest-stage3-amd64.txt
that contains two fields, the path to the file itself and a timestamp.

The timestamp is attempted to be included in the further downloading
automatically, resulting in 404 error. By splitting this properly
download works again
---
 templates/lxc-gentoo.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/templates/lxc-gentoo.in b/templates/lxc-gentoo.in
index d0df899..282522c 100644
--- a/templates/lxc-gentoo.in
+++ b/templates/lxc-gentoo.in
@@ -160,7 +160,7 @@ cache_stage3()
         printf "Determining path to latest Gentoo %s (%s) stage3 archive...\n" "${arch}" "${variant}"
         printf " => downloading and processing %s\n" "${stage3_pointer}"
 
-        local stage3_latest_tarball=$(wget -q -O - "${stage3_pointer}" | tail -n1 ) \
+        local stage3_latest_tarball=$(wget -q -O - "${stage3_pointer}" | tail -n1 | cut -d " " -f1) \
             || die 6 "Error: unable to fetch\n"
 
         printf " => Got: %s\n" "${stage3_latest_tarball}"
-- 
2.4.10

